### PR TITLE
quadrafuzz: init at 0.1.1

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by risicle** [comment]
Happily builds, non-nixos linux x86_64

**Comment by magnetophon** [comment]
updated the license as per https://github.com/jpcima/quadrafuzz/issues/1

**Comment by magnetophon** [comment]
Someone please merge this!  :)

File: pkgs/applications/audio/quadrafuzz/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, boost, cairo, lv2, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  pname = "quadrafuzz";
+  version = "unstable-17-10-2019";
+
+  src = fetchFromGitHub {
+    owner = "jpcima";
+    repo = pname;
+    rev = "a7cc7cba0bef09da9e6443e8967366650a372c37";
```
Comment (thiagokokada) [inline pkgs/applications/audio/quadrafuzz/default.nix:10]: Any reason for not using the tagged `0.1.0` release? I know this commit is newer than the tag, but I want to know if this is worth it.

File: pkgs/applications/audio/quadrafuzz/default.nix [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  makeFlags = [
+    "PREFIX=$(out)"
+  ];
+
+  installPhase = ''
+    mkdir -p $out/lib/lv2
+    cp -r bin/quadrafuzz.lv2/ $out/lib/lv2
+  '';
+
+  meta = with stdenv.lib; {
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/quadrafuzz/default.nix:34]:
```suggestion
  meta = with lib; {
```

File: pkgs/applications/audio/quadrafuzz/default.nix [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, boost, cairo, lv2, pkgconfig }:
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/quadrafuzz/default.nix:1]:
```suggestion
{ lib, fetchFromGitHub, boost, cairo, lv2, pkgconfig }:
```

File: pkgs/applications/audio/quadrafuzz/default.nix [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    mkdir -p $out/lib/lv2
+    cp -r bin/quadrafuzz.lv2/ $out/lib/lv2
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/jpcima/quadrafuzz;
+    description = "Multi-band fuzz distortion plugin";
+    maintainers = [ maintainers.magnetophon ];
+    platforms = platforms.linux;
+    license = licenses.gpl3;
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/quadrafuzz/default.nix:39]: gpl3Only or gpl3Plus?

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/applications/audio/quadrafuzz/default.nix (hunk 1 above)
Comment (magnetophon) [reply pkgs/applications/audio/quadrafuzz/default.nix:10]: @jpcima What's your take on this, latest commit or release?

File: pkgs/applications/audio/quadrafuzz/default.nix (hunk 4 above)
Comment (magnetophon) [reply pkgs/applications/audio/quadrafuzz/default.nix]: https://github.com/jpcima/quadrafuzz/blob/a7cc7cba0bef09da9e6443e8967366650a372c37/plugins/quadrafuzz/DistrhoPluginInfo.h#L32

File: pkgs/applications/audio/quadrafuzz/default.nix (hunk 1 above)
Comment (jpcima) [reply pkgs/applications/audio/quadrafuzz/default.nix:10]: I've released `0.1.1` just now, you can use this instead

Comment (magnetophon) [reply pkgs/applications/audio/quadrafuzz/default.nix:10]: Thanks, will do! :+1:

**Pull request merged**